rfq/manager.go

```diff
@@ -1066,9 +1067,11 @@ type ChannelWithSpecifier struct {
 // each asset channel that matches the provided asset specifier.
 func (m *Manager) ComputeChannelAssetBalance(ctx context.Context,
 	activeChannels []lndclient.ChannelInfo,
-	specifier asset.Specifier) ([]ChannelWithSpecifier, error) {
+	specifier asset.Specifier) (map[route.Vertex][]ChannelWithSpecifier,
```

Comment:
Style nit: can use a type def here to give the data structure a more descriptive name/type, and also cut down on the amt of chars needed to ref it a bit.